mobile_install/tools.bzl

```diff
@@ -91,7 +91,7 @@ TOOL_ATTRS = dict(
         executable = True,
     ),
     _d8 = attr.label(
-        default = Label("@bazel_tools//src/tools/android/java/com/google/devtools/build/android/r8:r8"),
```

Comment:
this change is invalid, but in general we have to break all the dependencies on `//external:android` from bazel, and this is one of them.